### ISSUE TITLE
Build error with Vaadin 14.1.2

### ISSUE THREAD

**Issue by javier-godoy** [FlowingCode/OrgChartAddon#14]
From thread in [Vaadin Directory](https://vaadin.com/directory/component/orgchart-add-on/discussions#replies17991164), Thomas Niedergesaess wrote: 

&ensp;I am using Vaadin 14.1.2.

&ensp;I am trying this component and getting following error:
```
...caused by: org.apache.maven.plugin.PluginExecutionException: Execution default of goal com.vaadin:vaadin-maven-plugin:14.1.2:build-frontend failed: 

  Failed to find the following css files in the `node_modules` or `/Users/thomas/TEST/GIT/my-server/src/BUSINESS_APP/frontend` directory tree:
      - orgchart/dist/css/jquery.orgchart.css
  Check that they exist or are installed. If you use a custom directory for your resource files instead of the default `frontend` folder then make sure it's correctly configured (e.g. set 'vaadin.frontend.frontend.folder' property)

	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:148)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:210)
	... 20 more
Caused by: java.lang.IllegalStateException: 
```

14.0.15: OK
14.1.x.: Error

**Comment by javier-godoy** [comment]
I couldn't reproduce this issue, but it should be fixed in version 4.0.2 of the addon.

**Comment by javier-godoy** [comment]
Confirmed that the component is working with Vaadin 14.1.3
